sycl/plugins/level_zero/pi_level_zero.cpp

```diff
@@ -3323,7 +3323,7 @@ pi_result piextQueueCreateWithNativeHandle(pi_native_handle NativeHandle,
       new _pi_queue(ZeQueues, ZeroCopyQueues, Context, Device, OwnNativeHandle);
   return PI_SUCCESS;
 }
-
+#if 0  // remove?
 // If indirect access tracking is enabled then performs reference counting,
 // otherwise just calls zeMemAllocDevice.
 static pi_result ZeDeviceMemAllocHelper(void **ResultPtr, pi_context Context,
```

Comment:
If we agree that you remove enableBufferPooling() switch in this patch ((i.e. make pooling enabled by default) ) then it is safe to remove this helper function.

Reply:
I retained the enableBufferPooling